io.github.phastmike.tags.json

```diff
@@ -0,0 +1,40 @@
+{
+    "app-id" : "io.github.phastmike.tags",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "tags",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "tags",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/phastmike/tags.git"
+                    "sha256": "1064b8222ea08afd1c111d98853240266dce475c"
```

Comment:
that doesn't work. I'm pretty sure flatpak-builder threw a warning. `shas256` isn't valid here.

Reply:
Oops, I'll check it

Reply:
changed sha256 to commit